src/main/java/net/juniper/netconf/NetconfSession.java

```diff
@@ -116,6 +117,11 @@ private void sendHello(String hello) throws IOException {
         String reply = getRpcReply(hello);
         serverCapability = reply;
         lastRpcReply = reply;
+        try {
+            serverHello = Hello.from(reply);
+        } catch (final Exception e) {
```

Comment:
is there are more specific exception that can be caught here?

Reply:
Yes, apologies, that was just me being lazy.